server/src/test/java/org/elasticsearch/index/mapper/AliasFieldTypeBooleanFieldTypeTests.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.elasticsearch.index.mapper;
+
+public class AliasFieldTypeBooleanFieldTypeTests extends BooleanFieldTypeTests {
+
```

Comment:
basically for all XXXFieldType tests i wanted to repeat them but this time using an alias to the "XXXFieldType". This way i can leverage all the BooleanFieldTypeTests, but this time both via the boolean field name and an alias.